.github/stale.yml

```diff
@@ -0,0 +1,40 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale. Set to `[]` to disable
+# We don't close any issue that is an enhancement or confirmed bug, but issues
+# waiting for reproduction cases and questions tend to get outdated.
+exemptLabels:
+  - "enhancement"
+  - "bug"
+  - "thinking"
+  - "docs"
+
+# Label to use when marking as stale
+staleLabel: "waiting-reply"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: |
+  Hey there,
+  We wanted to check in on this request since it has been inactive for at least 60 days.
+  If you think this is still an important issue in the latest version of [Consul](https://github.com/hashicorp/consul/blob/master/CHANGELOG.md)
+  or [its documentation](https://www.consul.io/docs) please reply with a comment here to let us know and we'll keep it open for investigation.
```

Comment:
It might be nice to say that commenting will also draw attention to the issue. Maybe something like "please reply with a comment here, which will cause it to stay open and move it back to the top of our triage list." 

Of course, if that's not how the process will work we shouldn't say it, but it would be nice to give people a feeling that commenting will draw attention to the issue, rather than just prevent it from disappearing.

Reply:
That is sadly probably not how it will work - but I can switch `we'll keep it open for investigation.` to ` which will cause it to stay open for investigation`